model/Core/Vocabularies/RelationshipType.md

```diff
@@ -56,4 +58,12 @@ TODO
 - testDependency: TODOdescription
 - testTool: TODOdescription
 - variant: TODOdescription
+- buildInputOf: Input to the build instance
+- buildOutputOf: Output of the build instance
+- buildConfigOf: Build configuration of the build instance
+- buildToolOf: Build tool used in the build instance
+- buildInvokedBy: Agent that invoked the build
+- buildOnBehalfOf: Person for which buildInvokedBy is acting on behalf of
```

Comment:
Clarify that this is "Actor". Also need to clarify constraints on each of the relationships.

Reply:
Why not properties for buildInvokedBy and buildOnBehalfOf?